05_syntax_lab/01.py

```diff
@@ -3,3 +3,14 @@
 the user and prints the largest one
 """
 
+largestNumber = int(0)
```

Comment:
Is 0 really the largest number you expect to read? What if a user types only negative numbers?